test/provision/k8s_install.sh

```diff
@@ -58,7 +70,6 @@ esac
 
 retry_function "apt-get update"
 retry_function "apt-get install --allow-downgrades -y \
-    llvm \
```

Comment:
I think we need recent llvm, should we also add repos from here?

https://apt.llvm.org/

Reply:
It's already installed  on the base image ;-)